packages/edit-post/src/editor.js

```diff
@@ -116,20 +116,22 @@ class Editor extends Component {
 				<EditPostSettings.Provider value={ settings }>
 					<SlotFillProvider>
 						<DropZoneProvider>
-							<EditorProvider
-								settings={ editorSettings }
-								post={ post }
-								initialEdits={ initialEdits }
-								useSubRegistry={ false }
-								{ ...props }
-							>
-								<ErrorBoundary onError={ onError }>
-									<EditorInitialization postId={ postId } />
-									<Layout />
-									<KeyboardShortcuts shortcuts={ preventEventDiscovery } />
-								</ErrorBoundary>
-								<PostLockedModal />
-							</EditorProvider>
+							<useViewportMatch.__experimentalWidthProvider value={ undefined }>
```

Comment:
Do we need to force that this provider be rendered in the application? Isn't it enough for `useContext` to use a default when there's no explicit context value?

It would make the hook easier to use in other applications. And, unless I'm mistaken, it should "just work" if this wrapper element were removed here.

Reply:
Hi @aduth, this file is not supposed to be merged. I just added this here to make it easier to test, one can just add value here and see if the value is applied and the editor reacts differently.

Reply:
> Hi @aduth, this file is not supposed to be merged.

But it was merged? 🤔

Reply:
Oh, I see #19075 now. Will review 👍

Reply:
Yes, I'm sorry for that I prepared the revert and missed the push :(